plugins/destination/file/client/transformer.go

```diff
@@ -0,0 +1,190 @@
+package client
+
+import "github.com/cloudquery/plugin-sdk/schema"
+
+func (c *Client) TransformBool(v *schema.Bool) interface{} {
```

Comment:
can these functions be only exported as `Transform(v schema.CQType)` that will do a type switch instead?